src/oaklib/cli.py

```diff
@@ -3038,7 +3042,10 @@ def termset_similarity(
     if not isinstance(impl, SemanticSimilarityInterface):
         raise NotImplementedError(f"Cannot execute this using {impl} of type {type(impl)}")
     if information_content_file:
-        impl.cached_information_content_map = load_information_content_map(information_content_file)
+        if isinstance(impl, SemSimianImplementation):
+            impl.custom_ic_map_path = information_content_file
+        else:
+            impl.cached_information_content_map = load_information_content_map(information_content_file)
```

Comment:
one possibility in future is to relieve client of the need for out of band knowledge about impl details. The generic SemSim interface could have a load_ic_map method, with the generic impl being to directly load, and the semsimian impl passing the path through.